src/Model/EditorGroups.rei

```diff
@@ -9,4 +9,6 @@ let getActiveEditorGroup: t => option(EditorGroup.t);
 let isActive: (t, EditorGroup.t) => bool;
 let isEmpty: (int, t) => bool;
 
-let reduce: (t, Actions.t) => t;
+let reduce: (~defaultFont: Service_Font.font, t, Actions.t) => t;
+
+let setBufferFont: (~bufferId: int, ~font: Service_Font.font, t) => t;
```

Comment:
This isn't used yet, but the plan is to leverage this in the `VimStoreConnector`'s `updater` method - when we get the command to switch to terminal-normal mode, we'd set the font for all editors related to that buffer to the terminal font.

It's a bit hacky (and not quite correct - if the user changes the terminal font, the editors in terminal normal mode won't get updated). The problem is the `Feature_Editor` doesn't know anything about terminals or terminal normal mode. We might need to add a way to have some Onivim-specific context (like, should it use the terminal font or editor font?) so that we can update it correctly.